CHANGES.rst

```diff
@@ -91,6 +91,11 @@ resample
   images that have contributed with flux to an output (resampled)
   pixel. [#7345]
 
+transforms
+----------
+
+- Require the asdf transform version to be at least 0.3.0 [#7352]
```

Comment:
Let's be very explicit here in terms of exactly what's being referenced. It should say "Add requirement for asdf-transform-schemas >= 0.3.0".  I think this should also go under a heading of "general" in the change log, because it's an installation/setup item.